genotype_api/api/endpoints/analyses.py

```diff
@@ -64,21 +52,13 @@ def delete_analysis(
     return JSONResponse(f"Deleted analysis: {analysis_id}", status_code=status.HTTP_200_OK)
 
 
-@router.post("/sequence", response_model=list[Analysis])
+@router.post("/sequence", response_model=list[AnalysisResponse])
 def upload_sequence_analysis(
     file: UploadFile = File(...),
     session: Session = Depends(get_session),
     current_user: User = Depends(get_active_user),
 ):
-    """Reading vcf file, creating and uploading sequence analyses and sample objects to db"""
-
-    file_name: Path = check_file(file_path=file.filename, extension=".vcf")
-    content = file.file.read().decode("utf-8")
-    sequence_analysis = SequenceAnalysis(vcf_file=content, source=str(file_name))
-    analyses: list[Analysis] = list(sequence_analysis.generate_analyses())
-    check_analyses_objects(session=session, analyses=analyses, analysis_type="sequence")
-    create_analyses_sample_objects(session=session, analyses=analyses)
-    for analysis in analyses:
-        analysis: Analysis = create_analysis(session=session, analysis=analysis)
-        refresh_sample_status(session=session, sample=analysis.sample)
+    """Reading vcf file, creating and uploading sequence analyses and sample objects to db."""
```

Comment:
```suggestion
    """Reading VCF file, creating and uploading sequence analyses and sample objects to the database."""
```
Unrelated, but could this route be a starting point for replacing the Genptype CLI upload?

Reply:
I think it actually is... if by upload pushing to the sql database is meant, then this would do it.

Reply:
Great! Then we have a starting point later.